### DISCUSSION TITLE
Spring Course Info?

### DISCUSSION THREAD

**Discussion by hanried** [earthlab-education/Earth-Analytics-AY24#659] [General]
Could someone provide some info on what to expect in the spring course?

I found this website with info about GEOG 4563/5563 but I'm not sure how up/outdated it is: https://www.earthdatascience.org/courses/earth-analytics-python/earth-analytics-syllabus/

Thanks in advance!

**Comment by akpost21** [comment]
@eculler @katherinesiegel Could you please provide more info about the spring course? Thanks!

**Reply by katherinesiegel** [reply]
@hanried The spring course will build on the fall course -- there will be some additional units and assignments, and then in the second half of the semester, we will pivot to focus on capstone projects
